### PR TITLE
영어, 한국어 STT 안내 메세지 API 링크 수정

### PR DESCRIPTION
**Pull request by kimtaerin66**

### CODE REVIEW

**Comment by kimtaerin66** [comment]
> I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

File: streamlit_frontend/speech_to_text.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def main():
         api_addr, _, _  = st.columns([2, 1, 1])
         
         api_addr.markdown(
-        f"<p style='display: inline-block;'><span style='display:inline-block;'><p>1. {translate('select_korean', st.session_state.ko_en)}</p>2. {translate('link_expl', st.session_state.ko_en)}<a style='display: inline-block; text-align: left;' href=https://developers.vito.ai/docs/>{translate('link_name', st.session_state.ko_en)}</a></p><p>3. {translate('play_wav', st.session_state.ko_en)}</p></span>",
+        f"<p style='display: inline-block;'><span style='display:inline-block;'><p>1. {translate('select_korean', st.session_state.ko_en)}</p>2. {translate('link_expl', st.session_state.ko_en)}<a style='display: inline-block; text-align: left;' href=https://platform.openai.com/docs/guides/speech-to-text>{translate('link_name', st.session_state.ko_en)}</a></p><p>3. {translate('play_wav', st.session_state.ko_en)}</p></span>",
```
Comment (MinDongRyul) [inline streamlit_frontend/speech_to_text.py:32]: API 링크가 잘못되었네요. 수정해주셔서 감사합니다! 고생하셨습니다!

**Review by MinDongRyul — APPROVED** [review]

**Comment by MinDongRyul** [comment]
> **CLA Assistant Lite bot:** Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/cla-assistant/github-action/blob/master/SAPCLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.
> 
> I have read the CLA Document and I hereby sign the CLA
> 
> **taerin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).You can retrigger this bot by commenting **recheck** in this Pull Request

I have read the CLA Document and I hereby sign the CLA

**Comment by kimtaerin66** [comment]
> I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

**Pull request merged**